tests/test_isrFunctions.py

```diff
@@ -344,7 +344,9 @@ def test_gainContext(self):
         mi = self.inputExp.getMaskedImage().clone()
         with ipIsr.gainContext(self.inputExp, self.inputExp.getImage(), apply=True):
             pass
-        self.assertMaskedImagesEqual(self.inputExp.getMaskedImage(), mi)
+        self.assertTrue(True)  # CZW
```

Comment:
What's this here for?

Reply:
This and the subsequent commented out test were the result of a bug in the isrMock code that I forgot to track down (the mock detector didn't properly rebuild the amplifiers to set the gain).  This is now fixed, and these two work arounds are removed as the test now completes correctly.